astropy/tests/helper.py

```diff
@@ -200,3 +201,68 @@ def assert_quantity_allclose(actual, desired, rtol=1.0e-7, atol=None, **kwargs):
     np.testing.assert_allclose(
         *_unquantify_allclose_arguments(actual, desired, rtol, atol), **kwargs
     )
+
+
+def assert_objects_equal(actual, desired, /, *, attrs=None, compare_class=True):
+    """Compare ndarray objects the attribute level.
+
+    Parameters
+    ----------
+    actual and desired: np.ndarray (positional only)
+
+    attrs: list[str] (optional, keyword only)
+       a list of attributes to compare in addition to the default list:
+       ['info.name', 'info.format', 'info.unit',
+        'info.description', 'info.meta', 'info.dtype']
+
+    compare_class: bool (default: False, keyword only)
+      whether to compare object classes via the equality operator (==)
+
+    Raises
+    ------
+    AssertionError
+      If actual doesn't compare equal to desired.
+    """
+    import numpy as np
```

Comment:
Just import numpy on top.

Reply:
I don't know, `assert_quantity_equal` does the import internally so there seem to be an intention to not import jumpy at the top level in this module.

Reply:
I think that is a mistake too - `numpy` is a required dependency, so there is no reason not to import on top.

Reply:
I did my homework and it looks like nobody had a clue last time this was discussed (6 years ago) either:
https://github.com/astropy/astropy/pull/7384/files#r182256821

if there was a reason, it seems to be lost now, so let's save future maintainers from wondering about that too and just move it now.